hooks/boot/09-version_locking.rb

```diff
@@ -1,4 +1,4 @@
-if system('command -v foreman-maintain > /dev/null')
+if foreman_maintain_installed?
```

Comment:
Thinking out loud: should this be:
```suggestion
if package_lock_feature?
```